server.js

```diff
@@ -21,7 +21,7 @@ app.get('*', (req, res) => {
   res.sendFile(path.join(__dirname, './spec/index.html'));
 });
 
-const port = 8080;
+const port = 8081;
```

Comment:
?

Reply:
it conflicts with our own port, makes life easier